src/main/java/com/youngcamp/server/controller/ExampleController.java

```diff
@@ -33,4 +35,12 @@ public SuccessResponse<String> getResource(@RequestParam(required = false) Strin
     String resourceData = "Resource with ID: " + id;
     return new SuccessResponse<>("Resource found", resourceData);
   }
+
+  @Autowired private AdminChecker adminChecker;
+
+  @GetMapping("/api/example/admincheck")
+  public String AdminCheckControl() {
+    if (adminChecker.isAdmin()) return "1";
+    else return "0";
+  }
```

Comment:
```suggestion
  public Boolean AdminCheckControl(){
   return adminChecker.isAdmin();
  }
```
string보다는 boolean이 더 명확할 것 같은데 이렇게 수정해보는 건 어떨까요?

Reply:
이거는 너무 좋습니다 !
좀 더 간결하고 좋을 것 같네요!! 👍 

바로 적용 해 두겠습니다!